For bookkeeping's sake:

The CI gave us errors on every job that was able to run on 2 different machines with:
```
In file included from /usr/workspace/umdev/gitlab-runner/umdev/builds/BXbynAgzP/001/gitlab/radiuss/Umpire/src/umpire/interface/c_fortran/wrapResourceManager.cpp:22:
In file included from /usr/workspace/umdev/gitlab-runner/umdev/builds/BXbynAgzP/001/gitlab/radiuss/Umpire/src/umpire/strategy/ResourceAwarePool.hpp:18:
/usr/workspace/umdev/gitlab-runner/umdev/builds/BXbynAgzP/001/gitlab/radiuss/Umpire/src/umpire/Umpire.hpp:77:10: error: invalid digit '9' in octal constant
  return UMPIRE_VERSION_MINOR;
         ^
/dev/shm/dane578-2916649/build_dane-toss_4_x86_64_ib-clang@14.0.6-krjqgkg5/include/umpire/config.hpp:41:31: note: expanded from macro 'UMPIRE_VERSION_MINOR'
#define UMPIRE_VERSION_MINOR 09
                              ^
1 error generated.
```

which matches what that external user said... so David is trying a fix.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -10,6 +10,7 @@ cmake_policy(SET CMP0025 NEW)
 cmake_policy(SET CMP0048 NEW)
 cmake_policy(SET CMP0057 NEW)
 cmake_policy(SET CMP0054 NEW)
+cmake_policy(SET CMP0096 OLD)
 
 include(CMakeDependentOption)
 include(CMakePackageConfigHelpers)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -4,6 +4,7 @@
 #
 # SPDX-License-Identifier: (MIT)
 ##############################################################################
+cmake_minimum_required(VERSION 3.23)
 
 cmake_policy(SET CMP0025 NEW)
 cmake_policy(SET CMP0048 NEW)
@@ -16,8 +17,6 @@ include(CMakePackageConfigHelpers)
 project(Umpire
   LANGUAGES CXX C
   VERSION 2025.09.0)
-
-cmake_minimum_required(VERSION 3.23)
 
 set(UMPIRE_VERSION_RC "")
 

